apps/aecore/src/aec_tx_pool.erl

```diff
@@ -743,6 +744,9 @@ check_tx_ttl(STx, _Hash, Height, _Event) ->
         false -> ok
     end.
 
+check_valid_at_protocol(STx, _Hash, Height, _Event) ->
```

Comment:
This makes the tx pool reject txs that is not valid yet. So you need to wait for the fork height before submitting your new tx. Probably not a problem, since there will be plenty other txs to mine in the first generations anyway.

Reply:
Yes, we might want to reconsider this at some point.